compiler/src/dmd/nogc.d

```diff
@@ -218,16 +264,32 @@ public:
 
 Expression checkGC(Scope* sc, Expression e)
 {
+    /* If betterC, allow GC to happen in non-CTFE code.
```

Comment:
```suggestion
    /* If betterC, allow GC to happen in CTFE code.
```

Reply:
The compiler cannot know in advance if a function will be used for CTFE or not, hence the comment is correct.